src/options/view.rs

```diff
@@ -364,12 +371,18 @@ impl TimeFormat {
                 //   - there is nothing after the first `\n`
                 // line 2 will be empty when:
                 //   - there exist at least 2 `\n`, and no content between the 1st and 2nd `\n`
-                let empty_recent_format_msg = "Custom timestamp format for recent files is empty, \
-                    please supply a chrono format string at the second line.";
+                let invalid_recent_format_msg =
+                    "Custom timestamp format for recent files is invalid, \
+                    please supply a valid chrono format string at the second line.";
                 let recent = lines.next().map(|rec| {
                     if rec.is_empty() {
-                        panic!("{}", empty_recent_format_msg)
+                        panic!("{}", invalid_recent_format_msg)
                     } else {
+                        StrftimeItems::new(rec).for_each(|item| {
+                            if matches!(item, Item::Error) {
+                                panic!("{}", invalid_recent_format_msg)
+                            }
+                        });
```

Comment:
This should include a comment explaining the motivation behind the code here, as e.g. described in the PR

Reply:
Also, seeing as this is present above as well, it may be worth deduplicating this